HACKING.md

```diff
@@ -106,6 +106,47 @@ otherwise an official image can be run with
 
 FAF should be available at ```http://localhost:8080/faf/```
 
+### Reporting into FAF
+1. Set up your server URL for libreport
+
+    Insert `URL = http://localhost:8080/faf` into `/etc/libreport/plugins/ureport.conf`
+2. Create crash
+
+    Make sure you have ABRT running and that you have autoreporting enabled (`abrt-auto-reporting enabled`)
+    Install package `will-crash` and then execute `will_abort`
+
+This should be enough, so in a few minutes (reports are processed every few minutes) you should see
+spike in your FAF. However there are a few places where it can go wrong, therefore following
```

Comment:
- s/you should see spike in your FAF/you should see a spike in graph on your FAF/
- s/there are a few places/there are few places/
- s/therefore following troubleshooting/therefore following troubleshooting might be required/